app/helpers/search_bar_helper.rb

```diff
@@ -13,6 +13,7 @@ def mount_search_bar
                         searchQuery: params[:search],
                         url: url,
                         useKeyShortcuts: true,
+                        isPF4: false,
```

Comment:
If it will be the default this is how it would look:

![gnome-shell-screenshot-928P80](https://user-images.githubusercontent.com/26363699/131486386-58b4dfbf-1c87-4ba2-a2ea-8903196a1235.png)

maybe it will give us a push with replacing buttons all over :)

Reply:
Maybe I missed something, why do we want the legacy pages to have a different style? IMHO one of the biggest strengths of the ruby helpers is that we can change everywhere at once and plugins don't need to care.

Reply:
because on old .erb pages the table and buttons on the page are still in bootstrap3 style,
I think it will just not look well in most of the pages until we switch those components to PF4 style,
when we'll do that, we can just tweak the `isPF4` prop.